docs/admin/install/eventing/install-eventing-with-yaml.md

```diff
@@ -32,19 +32,18 @@ To install the Eventing component:
     ```
```

Comment:
It's not a line you changed, but would you mind changing `--namespace` above to `-n` please for the sake of consistency?